pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -389,6 +390,9 @@ Indicates whether the master DNS port is open. An open master DNS port listens p
 *  `master_dns_bindall: 'true'` The master DNS port is open. This is the default value.
 *  `master_dns_bindall: 'false'` The master DNS port is closed.
 
+### master_external_loadbalancer
+The DNS name or IP address for the load balancer. If specified, this is included as subject alternative name in the [DC/OS certificate](/1.10/security/ent/tls-ssl/) of the host on which the load balancer is running.
```

Comment:
> of the host on which the load balancer is running.

This is not correct.

It makes *Admin Router on the master nodes* present an X.509 certificate including the subject alternative name corresponding to the IP address / DNS name of the level 4 load balancer in front of it (this is meant to be used for a non-TLS-terminating level **4** load balancer running on a host/hosts that is/are *not* part of DC/OS, in front of the master nodes, routing _external_ traffic to Admin Router on the master nodes).

Example is our AWS deployment. For example, `adyatlov-elasticl-17kqbudnuzk0y-682688633.us-west-2.elb.amazonaws.com` is a DNS name that represents a TCP level load balancer in front of the master nodes, whereas when the browser accesses `https://adyatlov-elasticl-17kqbudnuzk0y-68268863.....` the TLS handshake is directly performed with Admin Router running on the master node that the TCP connection was routed to.

Reply:
Thank you very much for the explanation. I have modified the text accordingly.